epoxy-annotations/build.gradle

```diff
@@ -1,7 +1,9 @@
 apply plugin: 'java'
 
-sourceCompatibility = 1.7
-targetCompatibility = 1.7
+apply from: "$rootDir/blessedDeps.gradle"
+
+sourceCompatibility = JavaVersion.VERSION_1_7
```

Comment:
Great catch, I forgot this was an android specific project. I left it for a per-module basis in case there was a module that wasn't a dependency of any android specific project. This would allow us the nice J8 features, but obviously in this project's context it is not needed. 👍